addons/web/static/src/js/core/rpc.js

```diff
@@ -48,7 +49,10 @@ return {
             params.model = options.model;
             params.method = options.method;
             params.kwargs = _.extend(params.kwargs || {}, options.kwargs);
-            params.kwargs.context = options.context || params.context || params.kwargs.context;
+            params.kwargs.context = options.context || params.context || params.kwargs.context || {};
```

Comment:
instead in session.js, around l308
```javascript
        // we add here the user context for ALL queries, because we can
        _.extend(params.kwargs.context, this.user_context);
```